src/main/java/pl/lodz/p/java/flyweight/model/Income.java

```diff
@@ -0,0 +1,14 @@
+package pl.lodz.p.java.flyweight.model;
+
+import java.util.Date;
+
+public class Income {
+
+    private IncomeType type;
+    private String name;
+    private double amount;
+    private boolean recurring;
+    private Date dateTime;
+    private Date availableFrom;
```

Comment:
Z naszej rozmowy
Agnieszka
powiedz mi jeszcze co miałeś na mysli z polami availableFrom i availableTo dla przychodów ?
Anton Lytvynovskyi
Anton
hmm...
Znaczy to jeszcze nie do konca przemyslalem
Myslialem pod kontem miesiacznego wynagrodzenia w firmie
I do tego myslalem dodac pole jeszcze recurring
Zeby jak zaznaczam, to mi jakos z automatu wchodzilo, ze np 1 dnia miesiaca dostaje wynagrodzenie"

wycofujemy sie z tego ?